modules/dkan/dkan_permissions/dkan_permissions.module

```diff
@@ -3,3 +3,27 @@
  * @file
  * Drupal needs this blank file.
  */
+ /**
+
+/*
+ * Implements hook_permission().
+ */
+function dkan_permissions_permission() {
```

Comment:
Sorry @jacintocapote I need to send this back to you one more time.  These functions really should be in dkan_datset_groups, not dkan_permissions.